typescript/cli/src/tests/commands/core.ts

```diff
@@ -1,117 +1,163 @@
 import { $, ProcessPromise } from 'zx';
 
 import { DerivedCoreConfig } from '@hyperlane-xyz/sdk';
-import { Address } from '@hyperlane-xyz/utils';
+import { ProtocolType } from '@hyperlane-xyz/utils';
```

Comment:
I'd keep the individual functions and have the enw class call them instead of completely remove them as they might be useful outside of the class context too, for example in warp e2e tests we sometimes save or read files from other paths. Having to create a class for using the methods seems a bit overkill in that case